crates/payload/basic/src/lib.rs

```diff
@@ -57,6 +57,10 @@ use tracing::{debug, trace};
 
 mod metrics;
 
+/// Ethereum payload builder
+#[cfg(not(feature = "optimism"))]
```

Comment:
```suggestion
#[derive(Debug,Clone, Default)
#[non_exhaustive]
```

Reply:
Added without `non_exhaustive` since it cannot be constructed outside of the defining crate with the attribute